.circleci/config.yml

```diff
@@ -3,18 +3,103 @@ version: 2.1
 orbs:
   snyk: snyk/snyk@0.0.8
 
+workflows:
+  integration:
+    jobs:
+      - checkout_code
+      - lint:
+          requires:
+            - checkout_code
+      - test:
+          requires:
+            - checkout_code
+      - security-scan:
+          requires:
+            - checkout_code
+      - build:
+          requires:
+            - lint
+            - test
+            - security-scan
+      - hold-for-deploy:
+          type: approval
+          filters:
+            branches:
+              only: master
+      - docker-push:
+          filters:
+            branches:
```

Comment:
Is there a tags filter?  Do we want a new build/push for each commit, or just for releases?

Reply:
Yup, looking to change that now.